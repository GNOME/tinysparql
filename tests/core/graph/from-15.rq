SELECT DISTINCT ?s ?l
FROM example:graphA
FROM example:graphB
WHERE {
  ?s rdf:type/rdfs:label ?l
}
ORDER BY ?l
