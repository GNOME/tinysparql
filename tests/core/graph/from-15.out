"http://example/resource"	"Bar"
"http://example/resource"	"Foo"
